### PR TITLE
Fix for #198 start-polaris throwing error

### PR DESCRIPTION
**Pull request by Tiberriver256**
I was unable to re-create the issue by running Start-Polaris and Stop-Polaris consecutively but if you happened to run Start-Polaris twice you were out of luck until restarting PowerShell as we were just creating a new instance of HttpListener right over the top of an already listening HttpListener and would throw an error.

@harikanth64 - If possible I'd love it if you could pull this code down and see if it resolves your issue before I merge it in.

### CODE REVIEW

**Comment by Tiberriver256** [comment]
@TylerLeonhardt - Got confirmation from @Harikanth64 this resolves this issue. When you get a chance could you give me a code review?

File: Tests/unit/PolarisServer.Tests.ps1 [hunk 1]
```diff
@@ -19,26 +19,23 @@ Describe "Test webserver use" {
...
-            if (-not $IsUnix) {
-                Stop-Polaris
-                $Polaris = Get-Polaris
-                $Polaris.Listener.IsListening | Should Be $false
-
-                $Polaris = Start-Polaris -Port 9998
-                $Polaris.Listener.IsListening | Should be $true
-            }
+
+        It "Should allow starting and stopping the server" {
```
Comment (TylerLeonhardt) [inline Tests/unit/PolarisServer.Tests.ps1]: I think we can probably just have one tests that stops/starts/stop/starts Polaris. It's typically best practice that single tests don't depend on something happening in another test.

Comment (Tiberriver256) [reply Tests/unit/PolarisServer.Tests.ps1]: I think I got your meaning. How does it look now?

Comment (TylerLeonhardt) [reply Tests/unit/PolarisServer.Tests.ps1]: Yes that looks good! Just wondering though... any reason you removed the "Allow a custom logger" test?

Comment (Tiberriver256) [reply Tests/unit/PolarisServer.Tests.ps1]: The exact reason would be that I am a dingbat... haha. Added the test back in, thanks for catching that.

**Pull request merged**